terra_notebook_utils/cli/drs.py

```diff
@@ -55,6 +55,20 @@ def drs_copy_batch(args: argparse.Namespace):
     args.workspace, args.google_billing_project = Config.resolve(args.workspace, args.google_billing_project)
     drs.copy_batch(args.drs_urls, args.dst, args.workspace, args.google_billing_project)
 
+@drs_cli.command("check-accessible", arguments={
+    "drs_url": dict(type=str),
+    ** workspace_args,
+})
+def drs_check_accessible(args: argparse.Namespace):
+    """
+    Fetch the first byte of a drs:// object to verify access
+    example:
+        tnu drs check-accessible drs://crouching-drs-hidden-access
+    """
+    args.workspace, args.google_billing_project = Config.resolve(args.workspace, args.google_billing_project)
+    result = drs.check_accessible(args.drs_url)
```

Comment:
Both `args.workspace`, and `args.google_billing_project` need to be passed into `check_accessible`, otherwise this will fail for requester pays buckets.